src/engine/strat_engine/backstore/device.rs

```diff
@@ -78,7 +78,9 @@ fn signature(device: &HashMap<String, String>) -> String {
 /// Determine what a block device is used for.
 pub fn identify(devnode: &Path) -> StratisResult<DevOwnership> {
     if let Some(device) = get_udev_block_device(devnode)? {
-        if empty(&device) {
+        if device.contains_key("DM_MULTIPATH_DEVICE_PATH") {
```

Comment:
For consistency it would make more sense to extend function ```empty``` to include a match on ```DM_MULTIPATH_DEVICE_PATH``` and to move this if block below the ```if empty``` block. This would make the code better consistent with the the code in ```get_all_empty_devices``` which is changed at line 63 to filter out multipath devices using ```DM_MULTIPATH_DEVICE_PATH```.